aicsimageio/tests/test_aics_image.py

```diff
@@ -248,7 +254,7 @@ def test_imread(resources_dir, filename, expected_shape):
         assert img.shape == expected_shape
 
         # Reshape and transpose are required so there should be two tasks in the graph
```

Comment:
this comment about two tasks seems invalid now

Reply:
Yea, it should be updated to "2 * chunks read", for CZI and LIF that chunk would be `2 * ZYX chunks read` and for TIFFs that chunk would be `2 * YX chunks read`

Reply:
Since the test LIF file is 2 channel, the `2 * chunks` still holds though